pages/linux/hashcat.md

```diff
@@ -0,0 +1,32 @@
+# hashcat
+
+> Fast and advanced password recovery tool.
+> More information: <https://hashcat.net/hashcat/>.
+
+- Perform a brute-force attack (mode 3) with default hashcat mask:
+
+`hashcat --hash-type {{id_hash_type}} --attack-mode 3 {{hash_value}}`
```

Comment:
It's the ID of the hash type, correct? I think `hash_type_id` or just `hash_type` would be clearer.

Reply:
ok, I prefer to use `hash_type_id` because the parameters expects a number which map an hash type . `hash_type` instead would make you think "MD5" is an accepted value (which is wrong)

Reply:
Sure